columnar/src/columnar/writer/value_index.rs

```diff
@@ -141,13 +130,13 @@ impl IndexBuilder for MultivaluedIndexBuilder {
 /// new index builder for every single column.
 #[derive(Default)]
 pub struct SpareIndexBuilders {
```

Comment:
I don't understand the name `SpareIndexBuilders`

Reply:
renamed